src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+		submitNoSamplesPane_1.setBounds(33, 161, 650, 32);
+		controlsPane.add(submitNoSamplesPane_1);
+		   
+        checkBoxProblems = new JCheckBox("Problems");
+        checkBoxProblems.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxProblems.setToolTipText("Select Problems");
+        checkBoxProblems.setSelected(true);
+        submitNoSamplesPane_1.add(checkBoxProblems);
+        
+        submitAllProblemsRadioButton = new JRadioButton("Submit All");
+        submitAllProblemsRadioButton.setSelected(true);
+        submitNoSamplesPane_1.add(submitAllProblemsRadioButton);
+        
+        submitSelectedProblemsRadioButton = new JRadioButton("Submit Selected");
+        submitNoSamplesPane_1.add(submitSelectedProblemsRadioButton);
+        
+        selectedProblemsLabel = new JLabel("None selected");
+        selectedProblemsLabel.setToolTipText("None selected");
+        submitNoSamplesPane_1.add(selectedProblemsLabel);
+        
+        selectProblemsButton = new JButton("Select");
+        selectProblemsButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                showProblemList();
+            }
+        });
+        selectProblemsButton.setToolTipText("Select Problems");
+        submitNoSamplesPane_1.add(selectProblemsButton);
+        
+        JPanel submitLanguages = new JPanel();
+        FlowLayout fl_submitLanguages = (FlowLayout) submitLanguages.getLayout();
+        fl_submitLanguages.setAlignment(FlowLayout.LEFT);
+        submitLanguages.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+        submitLanguages.setBounds(33, 191, 650, 32);
+        controlsPane.add(submitLanguages);
+        
+        checkBoxLanguage = new JCheckBox("Languages");
+        checkBoxLanguage.setVerticalAlignment(SwingConstants.TOP);
+        checkBoxLanguage.setToolTipText("Select Languages");
+        checkBoxLanguage.setSelected(true);
+        submitLanguages.add(checkBoxLanguage);
+        
+        submitAllLanguagesRadioButton = new JRadioButton("Submit All Languages");
+        submitAllLanguagesRadioButton.setSelected(true);
+        submitLanguages.add(submitAllLanguagesRadioButton);
+        
+        submitSelectedLanguagesRadioButton = new JRadioButton("Submit Selected");
+        submitLanguages.add(submitSelectedLanguagesRadioButton);
+        
+        selectedLanguagesLabel = new JLabel("None selected Lang?");
+        selectedLanguagesLabel.setToolTipText("None selected");
+        submitLanguages.add(selectedLanguagesLabel);
+        
+        selectLanguageButton = new JButton("Select");
+        selectLanguageButton.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                showLanguageList();
+            }
+        });
+        selectLanguageButton.setToolTipText("Select Problems");
```

Comment:
The "Select Language" button tooltip should say "Select Languages", not "Select Problems".

Reply:
I consider this an approval showstopper.